orttraining/orttraining/training_api/module.cc

```diff
@@ -169,7 +167,13 @@ Module::Module(const std::string& train_model_path_or_bytes,
   }
 
   train_sess_ = std::make_unique<onnxruntime::InferenceSession>(session_options, env);
-  ORT_THROW_IF_ERROR(train_sess_->Load(train_model_path_or_bytes));
+  ORT_ENFORCE(model_identifiers.train_model != "" || model_identifiers.train_model_data != nullptr,
+              "Training Session Creation failed. Either the train model path or train model data should be specified.");
+
+  ORT_THROW_IF_ERROR(model_identifiers.train_model_data == nullptr
+                         ? train_sess_->Load(model_identifiers.train_model)
+                         : train_sess_->Load(model_identifiers.train_model_data, model_identifiers.train_model_len));
```

Comment:
if load from model data (buffer), is that will only be CPU or it is also possible from a device memory(for exmaple GPU)?

Reply:
It will be CPU only. This is entire model proto which is being loaded.